Kitodo/src/main/java/org/kitodo/production/forms/ProcessForm.java

```diff
@@ -1170,4 +1170,17 @@ public String getProcessingEndOfLastTask(ProcessDTO processDTO) {
     public List<TaskDTO> getCurrentTasksForUser(ProcessDTO processDTO) {
         return ServiceManager.getProcessService().getCurrentTasksForUser(processDTO, ServiceManager.getUserService().getCurrentUser());
     }
+
+    /**
+     * Gets the amount of processes for the current filter
+     * @return amount of processes
+     */
+    public String getAmount() {
+        try {
+            return ServiceManager.getProcessService().count(ServiceManager.getProcessService().getQueryForFilter(isShowClosedProcesses(),isShowInactiveProjects(),filter)).toString();
```

Comment:
This line is probably too long for Codacy! (also spaces missing after commas separating parameters in `getQueryForFilter` call)